Makefile

```diff
@@ -430,8 +432,42 @@ e2e-deploy-gatekeeper: e2e-helm-install
     --set mutatingWebhookTimeoutSeconds=2 \
     --set auditInterval=0
 
+e2e-deploy-base-ratify: e2e-notaryv2-setup e2e-notation-leaf-cert-setup e2e-inlinecert-setup
+	docker build --progress=plain --no-cache \
+	-f ./httpserver/Dockerfile \
+	-t localbuild:test .
+	kind load docker-image --name kind localbuild:test
```

Comment:
given we now have deploy base-ratify, and deploy ratify, we can give a different tag to the local build just for clarify

Reply:
that's good call, updated tag to base-test for the base-ratify image.